samples/samples/async_client_sample.py

```diff
@@ -0,0 +1,637 @@
+# -*- coding: utf-8 -*-
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+This application demonstrates how to do basic asynchronous operations using Cloud Spanner.
+"""
+
+
+import asyncio
+from google.cloud.spanner_v1 import (
+    keyset,
+    transaction,
+    Client,
+    session
+)
+from google.cloud.spanner_v1.types import (
+    CreateSessionRequest,
+    BatchCreateSessionsRequest,
+    GetSessionRequest,
+    ListSessionsRequest,
+    DeleteSessionRequest,
+    ExecuteSqlRequest,
+    ExecuteBatchDmlRequest,
+    ReadRequest,
+    TransactionOptions,
+    BeginTransactionRequest,
+    CommitRequest,
+    RollbackRequest,
+    TransactionSelector,
+    PartitionQueryRequest,
+    PartitionReadRequest
+)
+from google.cloud.spanner_v1.services import spanner
+import tracemalloc
+import argparse
+import time
+
+
+# [ FUNCTION for creating async client and spanner client object]
+def get_async_client_and_associate_database(instance_id, database_id):
+    
+    async_client = spanner.SpannerAsyncClient()
+
```

Comment:
nit: here as well remove all the extra lines

Reply:
Done